pyspecProcScripts/phasing.py

```diff
@@ -339,6 +371,7 @@ def filter_ranges(B, A):
             d[direct, 1 : N_rising + 1]
             + d_rising[direct, ::-1]
         ) / 2
+    d *= 2    
```

Comment:
I might be imagining this but I thought for the FID slice we multiply by 2 to conserve something (energy?) since we cut the first half out in the line above?